server/src/main/java/com/objectcomputing/checkins/services/memberSkill/MemberSkillController.java

```diff
@@ -0,0 +1,116 @@
+package com.objectcomputing.checkins.services.memberSkill;
+
+import com.objectcomputing.checkins.services.role.RoleType;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Error;
+import io.micronaut.http.annotation.*;
+import io.micronaut.http.hateoas.JsonError;
+import io.micronaut.http.hateoas.Link;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+@Controller("/services/member-skill")
+@Secured(SecurityRule.IS_ANONYMOUS)
```

Comment:
I think this service should be `@Secured(SecurityRule.IS_AUTHENTICATED)`

Reply:
👍